### PR TITLE
Test exchange

### PR DESCRIPTION
**Pull request by RamiALBASHA**
Handled unit test of the `exchange.py` module

### CODE REVIEW

File: test/test_exchange.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
+from numpy import arange, linspace
+from pandas import Series, datetime
+from pytest import approx, fixture
```
[the remark this answers is not in the document]
Comment (RamiALBASHA) [reply test/test_exchange.py]: Ok, I'll make the necessary modifications.

File: test/test_exchange.py [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+@fixture()
+def leaf_local_weather():
+    return Series({'time': datetime(2012, 8, 1, 11),
+                   'Tac': 26.84,
+                   'hs': 43.77,
+                   'Rg': 300.28742,
+                   'u': 2.17,
+                   'Ca': 400,
+                   'Pa': 101.3,
+                   'PPFD': 663.035})
```
[the remark this answers is not in the document]
Comment (RamiALBASHA) [reply test/test_exchange.py]: Ok, I'll make the necessary modifications.

[the remark this answers is not in the document]
Comment (RamiALBASHA) [reply test/test_exchange.py]: @christian34 I removed the dependency on pytest but the ci still fails. I don't get it.

[the remark this answers is not in the document]
Comment (pradal) [reply test/test_exchange.py]: I have to look at it.
CI fails before the test and before anything related to hydroshoot.
I will investigate it next week.

[the remark this answers is not in the document]
Comment (RamiALBASHA) [reply test/test_exchange.py]: Hi all, is the CI problem solved ?

[the remark this answers is not in the document]
Comment (pradal) [reply test/test_exchange.py]: Yes. I solve it just now. In my lasy time ;)

[the remark this answers is not in the document]
Comment (RamiALBASHA) [reply test/test_exchange.py]: You still have lasy time ?!
Thanks @pradal

**Pull request merged**